packages/fluentui/react-northstar/src/components/Header/HeaderDescription.tsx

```diff
@@ -4,17 +4,17 @@ import * as React from 'react';
 import {
   childrenExist,
   createShorthandFactory,
-  UIComponent,
   UIComponentProps,
   ChildrenComponentProps,
   ContentComponentProps,
   commonPropTypes,
   ColorComponentProps,
   rtlTextContainer,
-  ShorthandFactory,
 } from '../../utils';
-
-import { WithAsProp, withSafeTypeForAs } from '../../types';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+import { WithAsProp, withSafeTypeForAs, FluentComponentStaticProps, ProviderContextPrepared } from '../../types';
+import { useTelemetry, getElementType, useUnhandledProps, useAccessibility, useStyles } from '@fluentui/react-bindings';
 
 export interface HeaderDescriptionProps
```

Comment:
Please update `Accessibility` type to use `never`

Reply:
Done